Movie/MovieTests/CinemaListScreenImageServiceTests.swift

```diff
@@ -0,0 +1,43 @@
+// CinemaListScreenImageServiceTests.swift
+// Copyright © Alexandr Grigorenko. All rights reserved.
+
+@testable import Movie
+import XCTest
+
+/// Тесты CinemaListScreenImageService
+final class CinemaListScreenImageServiceTests: XCTestCase {
+    // MARK: - Private properties
+
+    private var imageService: ImageServiceProtocol!
+    private var imageAPIService: ImageAPIServiceProtocol!
+    private var cacheService: CacheServiceProtocol!
+    private var proxy: Proxy!
+
+    // MARK: - Public methods
+
+    override func teardown() {
```

Comment:
```suggestion
    override func teardown() {
```
super надо в тело везде добавить по проекту